### DISCUSSION TITLE
How to change virtual-input.png?

### DISCUSSION THREAD

**Discussion by DzGithub2805** [luckydog7/Funkin-android#1491] [Q&A]
How to change it? 
![virtual-input](https://user-images.githubusercontent.com/83911327/123738603-4dcc5a00-d8cf-11eb-9ca7-f08a3acb57a6.png)
(This is zack's virtual imput) i want it for my mod

**Comment by HatsuneMiku-github** [comment]
Replace it with the other Virtual-input.png it might be in assets/images or assets/shared/images

**Reply by HatsuneMiku-github** [reply]
A new answer: you need the source code

**Comment by KlavierGayming** [comment]
Thanks to using a embedded BitmapData class for the graphic you can't.
